docs/02-membership.md

```diff
@@ -0,0 +1,350 @@
+
+![RelativeWeights](https://github.com/cheeky-gorilla/membership/assets/76262359/93a6f5a0-393e-46c8-8dfd-a51aa6864f46)
+
+## 2.6 Quotes
+
+We asked members why they think the Guild is important for Ethereum. Last updated May 21 2024.
+
+**Adrian Manning (Lighthouse)** - "Lighthouse is the Rust implementation of Ethereum Beacon Chain, a key component in Ethereum's transition to PoS Although Lighthouse is one of a few major production Eth2 clients, it's imperative that more than 1 production client is used in the space to avoid the chain collapsing if bugs/vulnerabilities are found in a single client. Protocol Guild supports the teams working on minority clients, which is an important initiative." <br><br>
+**Alex Stokes (Applied Research Group (ARG))** - "The Protocol Guild is an experiment in public goods funding that aims to align the incentives of core protocol contributors with the continued stewardship of the ecosystem by allowing these contributors to share in the value created by their hard work. This mechanism is important for the future of Ethereum as it ensures continuity of protocol direction, maintenance, and growth amongst a set of dedicated individuals who deeply care about realizing the collective vision that inspires everyone who interacts with our community." <br><br>
+**Alex Vlasov (TXRX)** - "Ethereum - and blockchains, in general - unveils tremendous opportunities and one of them is building a more fair world. Protocol Guild serves several purposes: 1. It's an instrument for public goods funding, which is often underrated 2. It's a research and an exploration of innovative social mechanisms 3. It aims to support Ethereum Protocol contributors" <br><br>
+**Ansgar Dietrichs (Consensus R&D (EF))** - "I strongly believe that not enshrining a contributor funding mechanism into the base chain was the correct choice for Ethereum (crucial for its credible neutrality). But in order to keep attracting top level talent to the core dev space, we need to experiment with out-of-protocol ways of funding. Protocol Guild is a very exciting project in that regard and hopefully a first step towards a sustainable funding structure." <br><br>
+**Anton Nashatyrev (TXRX)** - "I would really love to see Guild as an instrument for long term incentivisation of people who just want to develop Ethereum and don't want to worry about buying food for themselves tomorrow " <br><br>
+**Carl Beekhuizen (Consensus R&D (EF))** - "As researchers we spend a lot of time worrying about incentive compatibility. This guild nudges us towards a better incentive landscape for those working on the protocol." <br><br>
+**Core Contributor** - "A diversified community will certainly benefit the entire Ethereum ecosystem in the long-term. I believe protocol guild is the right place to be for attracting multi-dimensional talents from various areas of strength, which is an ideal way that helps to maintain a healthy eco-system for the days to come." <br><br>
```

Comment:
Should we use "Guild Member (Anonymous)" instead, to make it clear these are from PG people, and not any core contributor?